service/assessment/assessment.go

```diff
@@ -163,6 +175,17 @@
 		o(s)
 	}
 
+	// Default to an in-memory storage, if nothing was explicitly set
```

Comment:
I think WIthStorage option is missing

Reply:
Ah, this can be removed, this was a remnant from an old implementation attempt.